exercises/grains/src/test/java/GrainsTest.java

```diff
@@ -0,0 +1,88 @@
+import org.junit.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class GrainsTest {
+
+    @Test
+    public void countAtCell1() {
+        final BigInteger result = Grains.compute(1);
```

Comment:
Please avoid using `final` as specified in the [policies](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations).